tests/test_backend/test_alpaca.py

```diff
@@ -323,124 +325,124 @@ def historic_agg_data(size, *args, **kwargs):
                             'l': 223.25,
                             'v': 1001626,
-                            'd': 1535635800000},
+                            't': 1535635800000},
                            {'o': 223.6,
                             'c': 224.18,
                             'h': 224.19,
                             'l': 223.59,
                             'v': 362322,
-                            'd': 1535635860000},
+                            't': 1535635860000},
                            {'o': 224.18,
                             'c': 224.5,
                             'h': 224.53,
                             'l': 224.1799,
                             'v': 318951,
-                            'd': 1535635920000},
+                            't': 1535635920000},
                            {'o': 224.49,
                             'c': 224.18,
                             'h': 224.49,
                             'l': 224.07,
                             'v': 239954,
-                            'd': 1535635980000},
+                            't': 1535635980000},
                            {'o': 224.21,
                             'c': 224.1593,
                             'h': 224.25,
                             'l': 224.01,
                             'v': 248875,
-                            'd': 1535636040000},
+                            't': 1535636040000},
                            {'o': 224.16,
                             'c': 224.478,
                             'h': 224.49,
                             'l': 224.13,
                             'v': 199907,
-                            'd': 1535636100000},
+                            't': 1535636100000},
                            {'o': 224.47,
                             'c': 223.8065,
                             'h': 224.57,
                             'l': 223.8,
                             'v': 298215,
-                            'd': 1535636160000},
+                            't': 1535636160000},
                            {'o': 223.81,
                             'c': 223.61,
                             'h': 223.83,
                             'l': 223.55,
                             'v': 208515,
-                            'd': 1535636220000},
+                            't': 1535636220000},
                            {'o': 223.63,
                             'c': 223.4484,
                             'h': 223.82,
                             'l': 223.3208,
                             'v': 217942,
-                            'd': 1535636280000},
+                            't': 1535636280000},
                            {'o': 223.42,
                             'c': 223.0002,
                             'h': 223.44,
                             'l': 222.91,
                             'v': 301328,
-                            'd': 1535636340000}]})
+                            't': 1535636340000}]})
```

Comment:
You need to add tests around compat and non-compat behavior. Hard to know if it works as expected.

Reply:
Added a little coverage